values-schema.yaml

```diff
@@ -2387,6 +2358,30 @@ properties:
                 $ref: '#/definitions/resources'
               operator:
                 $ref: '#/definitions/resources'
+      kyverno:
+        additionalProperties: false
+        properties:
+          enabled:
+            type: boolean
+            default: false
+          _rawValues:
+            $ref: '#/definitions/rawValues'
+          mode:
```

Comment:
Is mode solely for setting HA ? 
If so, could you rename it to ha, with type boolean?